packages/adp-tooling/boilerplates/controller.ejs

```diff
@@ -0,0 +1,86 @@
+/*!
+ * OpenUI5
```

Comment:
Please use the `templates` folder for templates. Right now it is only used for project generation, therefore, please restructure it to have `templates/project` and `templates/rta` (or some other better name)